src/TraitsV2/TraitedClass.class.st

```diff
@@ -102,7 +96,11 @@ TraitedClass >> addAndClassifySelector: selector withMethod: compiledMethod inPr
 
 { #category : #'accessing method dictionary' }
```

Comment:
Shouldn't the deprecated methods be added as extensions from Deprecated70? 

Like this it's easier to remove at the beginning of Pharo 8.